config-connector/solutions/iam/kpt/member-iam/README.md

```diff
@@ -0,0 +1,84 @@
+Member IAM
+==================================================
+
+# NAME
+
+  member-iam
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (defaults to `compute.networkAdmin`) in the project.
+
+  You can also create any number of service accounts and grant them any number of roles by altering and applying the altered config YAMLs.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/member-iam member-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account 
+  (`cnrm-system@[PROJECT_ID].iam.gserviceaccount.com`) with the following 
+  roles in your desired project (it doesn't need to be the project where you 
+  installed Config Connector):
+
+  - roles/resourcemanager.projectIamAdmin
+  - roles/iam.serviceAccountAdmin
+
+# USAGE
+
+  Replace `${PROJECT_ID?}` with your desired project ID value from 
+  within this directory:
+
+  ```
+  kpt cfg set . project-id VALUE
+  ```
+
+  Once the project ID is set in the configs, simply apply the YAMLs:
```

Comment:
I feel like we should rephrase this to make it clear that it is the customer's option to do this when they feel ready. They may want to apply additional configuration.

Reply:
I think for getting onboard, we want to be more prescriptive. I treat it like quickstart guide. What do you think?

Reply:
I think it's appropriate to push the customer towards best practices wherever possible, but if we're too pushy, we also risk pushing the customer away from best practices. This is especially true in this situation, where we're engaged in a new initiative and the package manager we're using isn't stable yet.

Reply:
I think we can be prescriptive in areas where we are certain of best practice and less prescriptive where we are not.

Reply:
As we discussed offline, let's just leave this kind details at the comfort level of the solution dev and update them later if we receive any customer feedback.